### PR TITLE
Update hungarian layout

### PR DESCRIPTION
**Pull request by KnockoffFeline**
Added Í letter that was forgotten from v1.

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt [hunk 1]
```diff
@@ -16,7 +16,7 @@ import com.dessalines.thumbkey.utils.KeyboardMode
 import com.dessalines.thumbkey.utils.SwipeDirection
 import com.dessalines.thumbkey.utils.SwipeNWay
 
-val THUMBKEY_HU_V1_MAIN = KeyboardC(
+val THUMBKEY_HU_V2_MAIN = KeyboardC(
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt:19]: You can keep this as its original name.

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt [hunk 2]
```diff
@@ -502,8 +512,8 @@ val THUMBKEY_HU_V1_SHIFTED = KeyboardC(
     ),
 )
 
-val THUMBKEY_HU_V1_KEYBOARD_MODES: Map<KeyboardMode, KeyboardC> = mapOf(
-    KeyboardMode.MAIN to THUMBKEY_HU_V1_MAIN,
-    KeyboardMode.SHIFTED to THUMBKEY_HU_V1_SHIFTED,
+val THUMBKEY_HU_V2_KEYBOARD_MODES: Map<KeyboardMode, KeyboardC> = mapOf(
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt:515]: You can leave these lines the same as before.

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt [hunk 3]
```diff
@@ -256,7 +261,7 @@ val THUMBKEY_HU_V1_MAIN = KeyboardC(
     ),
 )
 
-val THUMBKEY_HU_V1_SHIFTED = KeyboardC(
+val THUMBKEY_EN_V4_SHIFTED = KeyboardC(
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt:264]: Still incorrect here.

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt (hunk 1 above)
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt:19]: You don't need to increase the version, leave the same as before.

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt (hunk 2 above)
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyHUv2.kt:515]: This whole block you can leave as it was before.

**Review by dessalines — CHANGES_REQUESTED** [review]

**Comment by KnockoffFeline** [comment]
I am sorry, I am new to Github. I would like to approve these changes, but it would not let me. How do I get the changes through, then?

**Comment by dessalines** [comment]
Make the changes that I requested above, then I'll see it. Watch a tutorial video on git.

**Pull request closed without merging**